src/app/pages/system/bootenv/bootpool-attach/bootpool-attach-form.component.ts

```diff
@@ -0,0 +1,90 @@
+import {
+  ChangeDetectionStrategy,
+  ChangeDetectorRef, Component,
+} from '@angular/core';
+import { Validators } from '@angular/forms';
+import { Router } from '@angular/router';
+import { FormBuilder } from '@ngneat/reactive-forms';
+import { UntilDestroy, untilDestroyed } from '@ngneat/until-destroy';
+import filesize from 'filesize';
+import { map } from 'rxjs/operators';
+import { JobState } from 'app/enums/job-state.enum';
+import { helptext_system_bootenv } from 'app/helptext/system/boot-env';
+import { FormErrorHandlerService } from 'app/pages/common/ix-forms/services/form-error-handler.service';
+import { DialogService, WebSocketService } from 'app/services';
+
+@UntilDestroy()
+@Component({
+  selector: 'bootpool-attach-form',
+  templateUrl: './bootpool-attach-form.component.html',
+  styleUrls: ['./bootpool-attach-form.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class BootPoolAttachFormComponent {
+  isFormLoading = false;
+  route_success: string[] = ['system', 'boot'];
+  route_cancel: string[] = ['system', 'boot', 'status'];
+
+  form = this.fb.group({
+    dev: ['', Validators.required],
+    expand: [false as boolean],
+  });
+
+  dev = {
+    fcName: 'dev',
+    label: helptext_system_bootenv.dev_placeholder,
+    tooltip: helptext_system_bootenv.dev_tooltip,
```

Comment:
New forms now expect strings to be translated and will not do translation themselves. You need to add `this.translate.instant` for labels and tooltips here and in another component.
Yes, it's incorrect in other forms we've refactored like localization settings.